docs/key-concepts.md

```diff
@@ -0,0 +1,122 @@
+# Key Concepts
+
+Observability and monitoring are crucial in modern software development and operations. It’s all about understanding, monitoring, and fixing complex systems using data collection, analysis, and visualization. When organizations use observability and monitoring effectively, they can spot and solve problems quickly, improve performance, and ensure their applications and infrastructure run smoothly.
+
+Observability and monitoring essentially mean keeping a close eye on how systems behave and perform using logs and metrics. These logs and metrics are generated by various components within the system, including applications, servers, databases, and networking devices. They serve as valuable sources of information for understanding system health, diagnosing issues, and optimizing performance.
```

Comment:
```suggestion
The most common type of Observability or Telemetry data that is used within various systems include logs, metrics and traces. These telemetry data are generated by various components within the system, including applications, servers, databases, and networking devices.
```

I think a lot of the information in here is the same as the first paragraph. Let's try and not be too repetitive with the information.